tests/foreman/ui_airgun/test_contentview.py

```diff
@@ -1571,6 +1569,210 @@ def test_positive_remove_cv_version_from_default_env(session, module_org):
         assert ENVIRONMENT not in cvv['Environments']
 
 
+@tier2
+def test_positive_remove_promoted_cv_version_from_default_env(session, module_org):
+    """Remove promoted content view version from Library environment
+
+    :id: a8649444-b063-4fb4-b932-a3fae7d4021d
+
+    :Steps:
+
+        1. Create a content view
+        2. Add a puppet module(s) to the content view
+        3. Publish the content view
+        4. Promote the content view version from Library -> DEV
```

Comment:
Sorry! I'm not sure when, where we are promoting this to Dev?

Reply:
`repo.setup_content_view(module_org.id)` perform publish and promote operations inside

Reply:
OK